DigitalOffice/Tests/FileServiceTests/Consumers/GetFileConsumerTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.FileService.Broker.Consumers;
+using LT.DigitalOffice.FileService.Database.Entities;
+using LT.DigitalOffice.FileService.Repositories.Interfaces;
+using LT.DigitalOffice.FileServiceUnitTests.UnitTestLibrary;
+using LT.DigitalOffice.Kernel.Broker;
+using MassTransit;
+using MassTransit.Testing;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.FileServiceUnitTests.Consumers
+{
+    internal class GetFileConsumerTests
+    {
+        private static byte[] contentByte = Convert.FromBase64String("RGlnaXRhbCBPZmA5Y2U=");
+        private readonly string contentString = Convert.ToBase64String(contentByte);
+        private readonly string extension = ".jpg";
+        private readonly string name = "File";
```

Comment:
Why not transfer initialization to setup  method?

Reply:
Fixed